main/create_simulated_body_model.py

```
@@ -0,0 +1,1085 @@
+        self._warning_label.setText(
+            'Warning: duplicate body ranges' if duplicates else ''
+        )
+
+    # ---- Rendering ----
+    def _update_image(self) -> None:
+        try:
+            img, meta = render_combined_model(self.sim_params, ignore_offset=True)
+            self._current_image = img
+            self._last_meta = meta
+            self._display_image()
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)